clang/include/clang/CConv/RewriteUtils.h

```diff
@@ -61,6 +61,7 @@ class DeclReplacementTempl : public DeclReplacement {
 typedef DeclReplacementTempl<VarDecl> VarDeclReplacement;
 typedef DeclReplacementTempl<ParmVarDecl> ParmVarDeclReplacement;
 typedef DeclReplacementTempl<FieldDecl> FieldDeclReplacement;
+typedef DeclReplacementTempl<Decl> DynDeclReplacement;
```

Comment:
Maybe rename to `AnyDeclReplacement` ? The `Dyn` is a little odd to parse.

Reply:
Actually it's not used, C++ subtyping didn't work the way I wanted it to.

Reply:
Could be though? See comment below.